packages/contentstack/package.json

```diff
@@ -61,6 +61,7 @@
     "@types/mkdirp": "^1.0.2",
     "@types/mocha": "^8.2.3",
     "@types/node": "^14.18.63",
+    "@types/semver": "^7.5.8",
```

Comment:
Bump the version, this will be a minor bump

Reply:
done